cob_generic_can/package.xml

```diff
@@ -22,8 +22,7 @@
   <run_depend>libpcan</run_depend>
   <run_depend>libntcan</run_depend>
 
-
   <export>
-    <cpp lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lcob_generic_can_peaksysusb -lcob_generic_can_peaksys -lcob_generic_can_esd" cflags="-I${prefix}/common/include"/>
+    <cpp cflags="-I${prefix}/common/include" lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lcob_generic_can_peaksysusb -lcob_generic_can_peaksys -lcob_generic_can_esd"/>
```

Comment:
Do we need those flags at anymore? I guess this is a leftover from `rosbuild`...